services/121-service/migration/1713363871246-remove-startedRegistration-state.ts

```diff
@@ -1,18 +1,31 @@
 import { MigrationInterface, QueryRunner } from 'typeorm';
 
+const registrationStatus = 'startedRegistration';
+
 export class RemoveStartedRegistrationState1713363871246
   implements MigrationInterface
 {
   name = 'RemoveStartedRegistrationState1713363871246';
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     console.time('RemoveStartedRegistrationState1713363871246');
+    await queryRunner.commitTransaction();
+
+    await queryRunner.query(`DELETE FROM "121-service".event_attribute WHERE "eventId" IN
+      (SELECT id FROM "121-service"."event" WHERE "registrationId" IN
+      (SELECT "id" FROM "121-service"."registration" WHERE "registrationStatus" = '${registrationStatus}'))`);
+
+    await queryRunner.query(`DELETE FROM "121-service"."event" WHERE "registrationId" IN
+      (SELECT "id" FROM "121-service"."registration" WHERE "registrationStatus" = '${registrationStatus}')`);
+
+    await queryRunner.query(`DELETE FROM "121-service".registration_data WHERE "registrationId" IN
+      (SELECT "id" FROM "121-service"."registration" WHERE "registrationStatus" = '${registrationStatus}')`);
 
     await queryRunner.query(
-      `DELETE FROM "121-service"."registration"
-             WHERE "registrationStatus" = 'startedRegistration'`,
+      `DELETE FROM "121-service"."registration" WHERE "registrationStatus" = '${registrationStatus}'`,
```

Comment:
@aberonni I had to do these changes for this migration.

Reply:
Thanks for sharing. Should we make these changes on a separate PR to get them merged? I wouldn't want to wait for this PR to be merged before fixing a potentially production-disruptive issue.

Reply:
Yes, Sure. Let me create a MR to main then.